modules/test/playwright/tests/notification-web/emailNotificationTemplate.spec.ts

```diff
@@ -184,13 +208,14 @@ test('can save recipients roles in email notification template', async ({
 		).toBeChecked();
 	}
 
-	// Clean up
-
-	await emailNotificationTemplatePage.backURLButton.click();
-
-	await notificationTemplatesPage.frontEndDatasetItemActions.click();
-
-	await notificationTemplatesPage.frontEndDatasetItemActionDelete.click();
+	await expect(
+		page
+			.frameLocator('iframe')
+			.first()
+			.getByText(
+				'[%CURRENT_USER_FIRST_NAME%][%ADDRESS_STREETADDRESSLINE1%]'
+			)
+	).toBeVisible();
```

Comment:
Hey @igor-franca, can you run this test without the browser? For some reason, this test only passes here if we have the browser running :thinking: